mc_automation_tools/validators/mapproxy_validator.py

```diff
@@ -57,7 +54,7 @@ def validate_layer_from_pycsw(
         links = self.extract_from_pycsw(pycsw_records)
```

Comment:
you can use "items" maybe better 
maybe like this 
 for product_type, product_links in links.items():
        for li, link_value in product_links.items():
           if li in ["WMS", "WMTS_KVP"]:
                links[product_type][li] += f"&token={token}"
            elif li == "WMTS_BASE":
                links[product_type][li] += f"/{product_id}-{product_type}"